kpi/tests/api/v2/test_api_assets.py

```diff
@@ -289,13 +289,30 @@ def test_restricted_access_to_version(self):
         self.client.logout()
         self.client.login(username='anotheruser', password='anotheruser')
         resp = self.client.get(self.version_list_url, format='json')
-        self.assertEqual(resp.data['count'], 0)
+        self.assertEqual(resp.status_code, status.HTTP_404_NOT_FOUND)
         _version_detail_url = reverse(self._get_endpoint('asset-version-detail'),
                                       args=(self.asset.uid, self.version.uid))
         resp2 = self.client.get(_version_detail_url)
         self.assertEqual(resp2.status_code, status.HTTP_404_NOT_FOUND)
         self.assertEqual(resp2.data['detail'], 'Not found.')
 
+    def test_view_access_to_version(self):
+        """
+        Test user with submissions permissions can view versions list
+        """
+        self.client.logout()
+        self.client.login(username='anotheruser', password='anotheruser')
+        user = User.objects.get(username='anotheruser')
+        self.asset.assign_perm(user, PERM_VIEW_SUBMISSIONS)
+        resp = self.client.get(self.version_list_url, format='json')
+        self.assertEqual(resp.status_code, status.HTTP_200_OK)
+        self.assertEqual(resp.data['count'], 1)
+        _version_detail_url = reverse(self._get_endpoint('asset-version-detail'),
+                                      args=(self.asset.uid, self.version.uid))
+        resp2 = self.client.get(_version_detail_url)
+        self.assertEqual(resp2.status_code, status.HTTP_200_OK)
+        self.assertEqual(resp2.data.get('uid'), self.version.uid)
+
```

Comment:
Please add tests with anonymous user and public shared asset (with regular user and anonymous user). 
I think it would be great to add tests to validate the endpoint is readonly (e.g.: user with write permission on assets cannot do anything on versions. Neither `DELETE` or `PATCH`)